understanding/22/accessible-authentication-minimum.html

```diff
@@ -68,16 +68,17 @@ <h2>In brief</h2>
     <section id="intent">
         <h2>Intent of Accessible Authentication</h2>
 
-        <p>The purpose of this Success Criterion is to ensure there is an accessible, easy-to-use, and secure method to log in. Most Web sites rely on usernames and passwords for logging in. Memorizing or transcribing a username, password, or one-time verification code places a very high or impossible burden upon people with certain cognitive disabilities.</p>
+        <p>The purpose of this Success Criterion is to ensure there is an accessible, easy-to-use, and secure method for users to authenticate -- to prove they are authentic account holders. Most Web sites rely on usernames and passwords for logging in, the most common form of authentication. Yet memorizing or transcribing a username, password, or one-time verification code places a very high or impossible burden upon people with certain cognitive disabilities.</p>
```

Comment:
should also mention logic puzzles (including object recognition). this would also provide a logical segue to the next paragraph